lambify/Dockerfile

```diff
@@ -1,5 +1,15 @@
 FROM public.ecr.aws/lambda/ruby:2.7
 ARG RAILS_ENV
 ENV RAILS_ENV=$RAILS_ENV
+
 COPY . .
+
+# == Cleanup Unused Files & Directories ==
+RUN rm -rf \
+    log \
+    node_modules \
+    test \
+    tmp \
+    vendor/bundle/ruby/2.7.0/cache
```

Comment:
In order to support caching cleanly for the deploy step, we needed to move the `./bin/_build` cleanup to the Dockerfile after the working directory is copied to the container. I know this creates a "layer" but I'm not really worried about it and appreciate the speed bump. Feedback welcome.